Makefile

```diff
@@ -21,6 +21,7 @@ plugins: libs
 	$(MAKE) all -C plugins/Meters
 	$(MAKE) all -C plugins/Parameters
 	$(MAKE) all -C plugins/States
+	$(MAKE) all -C plugins/Super_Saw
```

Comment:
We are using uppercase and lowercase names here, so please use CamelCase for the folder name.
(ie, rename it SuperSaw)